CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 ## [0.11.2] (Unreleased)
 - Fix bug where generated code would not compile if a message was named Unit (#1126)
 - Add option to block generation/access to field getters (#1098)
-- 
+- Add option for enum annotation (#1155)
```

Comment:
Our changelog is out of date. This should go into a new "0.11.4 (Unreleased)" section.